ui/src/wizard/AzureWizard.ts

```diff
@@ -72,16 +82,26 @@ export class AzureWizard<T extends IInternalActionContext> implements types.Azur
                 if (step.shouldPrompt(this._context)) {
                     step.propertiesBeforePrompt = Object.keys(this._context).filter(k => !isNullOrUndefined(this._context[k]));
 
+                    const loadingQuickPick = this._showLoadingPrompt ? createQuickPick(this._context, {
+                        loadingPlaceHolder: localize('loading', 'Loading...')
+                    }) : undefined;
+
+                    loadingQuickPick?.onDidHide(() => {
```

Comment:
This returns a disposable that should be handled somehow. Also I can't remember - is `onDidHide` only called when the user escapes the pick or is it also called when a new pick is displayed and causes the old one to hide?

Reply:
> is onDidHide only called when the user escapes the pick or is it also called when a new pick is displayed and causes the old one to hide?

It's called on both, user escape and when a new pick is displayed.

Reply:
So isn't that a problem? I would think a new pick being displayed would cause the wizard to eventually get cancelled because of the token.

Reply:
Yeah it's a problem. It gets cancelled when it gets to the next step. Thinking of solutions 🤔